docs/development.rst

```diff
@@ -58,15 +82,83 @@ Rebuild and install from Python wheel package::
 
 
 To find out whether the created wheel package passes the `twine check` test and
-can be uploaded to PyPi run::
+can be uploaded to PyPi run
+
+.. code-block:: text
 
    make build test
 
 
 .. _venv: https://docs.python.org/3/library/venv.html
 
 
-Example pages
+Windows, or systems without ``make`` installed
+==============================================
+
+Windows does not have ``make`` therefore we must run the commands directly
+rather than using the shortcuts in the Makefile. Assume the commands below are
+all run in PowerShell. These instructions will also work on Mac or Linux without
+make installed as well.
+
+First, be sure to install Python 3, and Node 16.
+`nvm <https://github.com/coreybutler/nvm-windows>`_ is really useful for
+managing multiple versions of Node on Windows.
+
+Make a Python virtual environment. Let's make it in a folder called ``.venv``
+which will be ignored by git.
+
+.. code-block:: shell
+
+   # Create the venv
+   python -m venv ./.venv/
+
+   # Activate it (PowerShell)
+   ./.venv/Scripts/Activate.ps1
```

Comment:
I doubt this command would be identical on Mac or Linux?
(Presumably this'd be `. .venv/bin/activate`?)